app/src/main/java/com/example/campusguide/directions/TransitRouteAdapter.kt

```diff
@@ -0,0 +1,142 @@
+package com.example.campusguide.directions
+
+import android.app.Activity
+import android.graphics.Color
+import android.graphics.Typeface
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.BaseAdapter
+import android.widget.ImageView
+import android.widget.LinearLayout
+import android.widget.TextView
+import com.example.campusguide.R
+import java.text.SimpleDateFormat
+import java.util.Calendar
+import java.util.Locale
+
+class TransitRouteAdapter constructor(private val activity: Activity) : BaseAdapter() {
```

Comment:
A Javadoc to explain what the class does would be helpful i think, esp cause it's not always obvious what BaseAdapter is for.

Reply:
Agreed, done